upup/pkg/fi/cloudup/dotasks/droplet.go

```diff
@@ -29,23 +29,25 @@ import (
 )
 
 //go:generate fitask -type=Droplet
+// Droplet represents a group of droplets. In the future it
+// will be managed by the Machines API
 type Droplet struct {
 	Name      *string
-	ID        *string
 	Lifecycle *fi.Lifecycle
 
 	Region   *string
 	Size     *string
 	Image    *string
 	SSHKey   *string
 	Tags     []string
+	Count    int
```

Comment:
Was contemplating if I should use `*int` here instead. I vaguely remember having to use pointers tasks based on the way the tasks are rendered/compared but wasn't 100% sure. Will use just `int` since it seems to work as expected.

Reply:
Sounds good.  This field isn't part of our API, so it's easy to change anyway.

A few differences: *int can express `nil` as different from `0`.  The kops delta algorithm will by default treat `nil` in the expected value as meaning "don't care" (i.e. would not trigger a Render), whereas a `0` would be "must be zero".

The other difference is on the `changes` side; a `0` value in an `int` could mean "expected was zero, actual was non-zero" or it could mean "no change detected".  That ambiguity isn't present with `*int`.

But the TLDR: if you don't need to tell the difference, `int` is fine, and it's easy to change if you change your mind.